src/main/java/com/faforever/client/main/MainController.java

```diff
@@ -338,6 +339,20 @@ public void display() {
     registerWindowListeners();
   }
 
+  private void hideSplashScreen() {
+    try {
+      Class splashScreenClass = Class.forName("com.install4j.api.launcher.SplashScreen");
+      Method hideMethod = splashScreenClass.getDeclaredMethod("hide");
+      hideMethod.invoke(null);
+    } catch (ClassNotFoundException e) {
```

Comment:
Right, this is the case we want: Catching the case of no available install4j. So only debug message.

Reply:
Is this a confirmation that I put enough thought into this? ;)

Reply:
Yes :D .